packages/tsconfig.json

```diff
@@ -9,7 +9,10 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "downlevelIteration": true,
-    "traceResolution": false
+    "traceResolution": false,
+    "paths": {
+      "@hicommonwealth/*": ["../libs/*/src/index.ts"]
```

Comment:
Same issue with wildcard here.

Wouldn't this mean that every module now imports every other module?

Reply:
no, this is just to find the paths in dev mode ... a generic rule to avoid maintaining another list when adding more libs. 

references and imports will take care of dependencies

Reply:
once we migrate to pnpm and turborep, we might revisit this topic